### ISSUE TITLE
Compute node startup/respawn

### ISSUE THREAD

**Issue by kelvich** [zenithdb/zenith#114]
There are few tasks that we need to solve before we can restart compute node without any bad consequences.

Namely:
- download pg_controldata file
- download clog files
- download multixact files
- download 2pc files
- unlock auth #113

**Comment by kelvich** [comment]
Beside auth all other points are duplicate of https://github.com/zenithdb/zenith/issues/40
